integration-tests/keycloak-authorization/src/main/resources/application.properties

```diff
@@ -59,22 +59,24 @@ quarkus.keycloak.policy-enforcer.paths.10.enforcement-mode=ENFORCING
 quarkus.keycloak.policy-enforcer.paths.11.path=/api/public-token
 quarkus.keycloak.policy-enforcer.paths.11.enforcement-mode=DISABLED
 
-# Tenant
-
+# Service Tenant
 quarkus.oidc.tenant.auth-server-url=${keycloak.url}/realms/quarkus
 quarkus.oidc.tenant.client-id=quarkus-app
 quarkus.oidc.tenant.credentials.secret=secret
 
-# Enable Policy Enforcement
-quarkus.keycloak.tenant.policy-enforcer.enforcement-mode=PERMISSIVE
-
-# Defines a global claim to be sent to Keycloak when evaluating permissions for any requesting coming to the application
-quarkus.keycloak.tenant.policy-enforcer.claim-information-point.claims.request-uri={request.relativePath}
-quarkus.keycloak.tenant.policy-enforcer.claim-information-point.claims.request-method={request.method}
-
-# Defines a static claim that is only sent to Keycloak when evaluating permissions for a specific path
 quarkus.keycloak.tenant.policy-enforcer.paths.1.name=Permission Resource Tenant
 quarkus.keycloak.tenant.policy-enforcer.paths.1.path=/api-permission-tenant
 quarkus.keycloak.tenant.policy-enforcer.paths.1.claim-information-point.claims.static-claim=static-claim
 
+# Web App Tenant
+quarkus.oidc.webapp-tenant.auth-server-url=${keycloak.url}/realms/quarkus
+quarkus.oidc.webapp-tenant.client-id=quarkus-app
+quarkus.oidc.webapp-tenant.credentials.secret=secret
+quarkus.oidc.webapp-tenant.application-type=web-app
+quarkus.oidc.webapp-tenant.roles.source=accesstoken
+
+quarkus.keycloak.webapp-tenant.policy-enforcer.paths.1.name=Permission Resource WebApp
+quarkus.keycloak.webapp-tenant.policy-enforcer.paths.1.path=/api-permission-webapp
```

Comment:
Did you define a `/api-permission-webapp` endpoint?

Reply:
@pedroigor My fault, it was there but I just forgot to add a test resource :-). But yes, it is still failing

Reply:
@pedroigor I'll debug a bit more

Reply:
@pedroigor I fixed it, the endpoint protected by `web-app` had to have `@Authenticated` added - without it the initial HTTP request without a token was considered anonymous